src/app/utils/DiscordUtils.ts

```diff
@@ -68,6 +107,38 @@ const DiscordUtils = {
 		return messageText;
 	},
 
+    // TODO: graceful timeout handling needed
+    async awaitUser(channel: TextChannel, replyOptions: AwaitMessagesOptions): Promise<Message> {
+        let messages: Collection<Snowflake, Message> = null;
+        try {
+            messages = await channel.awaitMessages(replyOptions);
+            // TODO: this is too broad
+            } catch (e) {
+                throw new ValidationError(
+                    'You have timed out!\n' +
+                    'You can run `/bounty create` to create a new bounty. Please respond to my questions within 5 minutes.\n' +
+                    'Please reach out to your favorite Bounty Board representative with any questions.\n'
+                );
+        }
+         return messages.first();
+    },
+
+    // Send a response to a command (use ephemeral) or a reaction (use DM)
+    async actionResponse(commandContext: CommandContext, content: string, toUser: GuildMember): Promise<void> {
+        if (!!commandContext) { // This was a slash command
+            await commandContext.send({ content: content, ephemeral: true });
+            // await commandContext.delete();
+        } else {  // This was a reaction or a DB event
+            await toUser.send(content);
+        }
+    },
+
+    // Send a notification to an interested party (use a DM)
```

Comment:
This is also a good call - will allow us to change the notification style across all activities in one place. Helpful when we have threads on the horizon

nit: I would consider renaming the use of `action` in this method and the one above to `activity`